tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -85,13 +88,58 @@ def test_fsx_lustre_configuration_options(
         storage_type=storage_type,
         drive_cache_type=drive_cache_type,
         storage_capacity=storage_capacity,
+        imported_file_chunk_size=imported_file_chunk_size,
+        weekly_maintenance_start_time=weekly_maintenance_start_time,
     )
     cluster = clusters_factory(cluster_config)
-    _test_fsx_lustre(
-        cluster, region, scheduler, os, mount_dir, bucket_name, storage_type, auto_import_policy, deployment_type
+    _test_fsx_lustre_configuration_options(
+        cluster,
+        region,
+        scheduler,
+        os,
+        mount_dir,
+        bucket_name,
+        storage_type,
+        auto_import_policy,
+        deployment_type,
+        weekly_maintenance_start_time,
+        imported_file_chunk_size,
+        storage_capacity,
     )
 
 
+def _test_fsx_lustre_configuration_options(
+    cluster,
+    region,
+    scheduler,
+    os,
+    mount_dir,
+    bucket_name,
+    storage_type,
+    auto_import_policy,
+    deployment_type,
+    weekly_maintenance_start_time,
+    imported_file_chunk_size,
+    storage_capacity,
+):
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    scheduler_commands = get_scheduler_commands(scheduler, remote_command_executor)
+    fsx_fs_id = get_fsx_fs_id(cluster, region)
+    fsx = boto3.client("fsx", region_name=region).describe_file_systems(FileSystemIds=[fsx_fs_id])
+
+    _test_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir, os, region, fsx_fs_id)
+    _test_import_path(remote_command_executor, mount_dir)
+    _test_fsx_lustre_correctly_shared(scheduler_commands, remote_command_executor, mount_dir)
```

Comment:
Minor: instead of rewriting these common test cases can we just call `_test_fsx_lustre`?

Reply:
Done